diff -r -u cmake-2.8.12-src/Tests/CMakeLists.txt cmake-2.8.12-src_/Tests/CMakeLists.txt
--- cmake-2.8.12-src/Tests/CMakeLists.txt	2013-10-07 14:28:26.000000000 +0200
+++ cmake-2.8.12-src_/Tests/CMakeLists.txt	2020-07-18 15:28:09.084285368 +0200
@@ -1013,6 +1013,7 @@
     )
   list(APPEND TEST_BUILD_DIRS "${CMake_BINARY_DIR}/Tests/QtAutomocNoQt")
 
+  if(BUILD_QtDialog)
   find_package(Qt5Widgets QUIET NO_MODULE)
   if(Qt5Widgets_FOUND)
     add_test(Qt5Automoc ${CMAKE_CTEST_COMMAND}
@@ -1069,6 +1070,7 @@
       list(APPEND TEST_BUILD_DIRS "${CMake_BINARY_DIR}/Tests/Qt4And5Automoc")
     endif()
   endif()
+  endif()
 
   add_test(ExternalProject ${CMAKE_CTEST_COMMAND}
     --build-and-test
diff -r -u cmake-2.8.12-src/Tests/RunCMake/CMakeLists.txt cmake-2.8.12-src_/Tests/RunCMake/CMakeLists.txt
--- cmake-2.8.12-src/Tests/RunCMake/CMakeLists.txt	2013-10-07 14:28:26.000000000 +0200
+++ cmake-2.8.12-src_/Tests/RunCMake/CMakeLists.txt	2020-07-18 15:27:40.200319319 +0200
@@ -101,6 +101,8 @@
 add_RunCMake_test(TargetPolicies)
 add_RunCMake_test(alias_targets)
 
+
+if(BUILD_QtDialog)
 find_package(Qt4 QUIET)
 find_package(Qt5Core QUIET)
 if (QT4_FOUND AND Qt5Core_FOUND AND NOT Qt5Core_VERSION VERSION_LESS 5.1.0)
@@ -115,6 +117,7 @@
   add_RunCMake_test(include_external_msproject)
   add_RunCMake_test(SolutionGlobalSections)
 endif()
+endif()
 
 add_RunCMake_test(File_Generate)
 add_RunCMake_test(ExportWithoutLanguage)
